components/script/dom/servoparser/mod.rs

```diff
@@ -140,6 +137,13 @@ pub(crate) struct ServoParser {
     aborted: Cell<bool>,
     /// <https://html.spec.whatwg.org/multipage/#script-created-parser>
     script_created_parser: bool,
+    /// A decoder exclusively for input to the prefetch tokenizer.
+    ///
+    /// Unlike the actual decoder, this one takes a best guess at the encoding and starts
+    /// decoding immediately.
+    #[ignore_malloc_size_of = "Defined in tendril"]
```

Comment:
Set this to be sized in zero in `components/malloc_size_of`, so that if `tendril` is every updated we don't have to hunt down all of these types in the code.